src/Core/src/Handlers/SearchBar/SearchBarHandler.Android.cs

```diff
@@ -114,6 +114,17 @@ public static void MapCancelButtonColor(ISearchBarHandler handler, ISearchBar se
 			handler.PlatformView?.UpdateCancelButtonColor(searchBar);
 		}
 
+		public static void MapKeyboard(ISearchBarHandler handler, ISearchBar searchBar)
+		{
+			handler.PlatformView?.UpdateKeyboard(searchBar);
+		}
+
+		static void MapFocus(ISearchBarHandler handler, ISearchBar searchBar, object? args)
+		{
+			if (args is FocusRequest request)
+				handler.QueryEditor?.Focus(request);
+		}
+
```

Comment:
Is this being used in this PR? If not, can we remove it? (It'll be added via https://github.com/dotnet/maui/pull/13908 either way)

Reply:
Good catch. Removed.